### PR TITLE
cargo_c, bump version

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: dev-rust/cargo_c/cargo_c_bin-0.9.27.recipe [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	CHECKSUM_SHA256="39da8945db1ee54f051ea69b04d2c0a9c765baffab2c4871427c1d1f96b29e03"
+	SOURCE_DIR="cargo-c-$srcGitRev/32bit"
+elif [ "$targetArchitecture" = x86_64 ]; then
+	srcGitRev="a703bd97a18326396b7805aa75a6444796fe4605"
+	SOURCE_URI="https://github.com/Begasus/cargo-c/archive/$srcGitRev.tar.gz"
+	CHECKSUM_SHA256="0af0cef06db1aa7d46ca6ce60ad89bc1ca0c3705c51a7f80f6f87e0b2b84d8f7"
+	SOURCE_DIR="cargo-c-$srcGitRev/64bit"
+fi
+
+ARCHITECTURES="all !x86_gcc2"
```
Comment (korli) [inline dev-rust/cargo_c/cargo_c_bin-0.9.27.recipe:21]: Shouldn't it be x86_64 !all

**Comment by korli** [comment]
Also add DISABLE_SOURCE_PACKAGE="yes"

**Comment by Begasus** [comment]
Thanks, will update. :)

File: dev-rust/cargo_c/cargo_c_bin-0.9.27.recipe (hunk 1 above)
Comment (Begasus) [reply dev-rust/cargo_c/cargo_c_bin-0.9.27.recipe:21]: Actually should be "!all x86_64" then (otherwise !all will take over from x86_64)

Comment (jmairboeck) [reply dev-rust/cargo_c/cargo_c_bin-0.9.27.recipe:21]: or remove the `!all` part altogether and just use `"x86_64"`

**Pull request merged**